### PR TITLE
Prevent staff users from seeing other committees besides from there own.

### PR DESCRIPTION
**Pull request by cdconn00**
Fixes FM-29

### CODE REVIEW

File: app/controllers/committees_controller.rb [hunk 1]
```diff
@@ -24,7 +24,13 @@ def index
...
 
     @committee_rows = []
-    committee_list = Committee.all
+    committee_list = []
+
+    if current_user.check_staff? && !current_user.check_executive?
+      committee_list.push Committee.find_by(:name => current_user.committee.name)
+    else
+      committee_list = Committee.all
+    end
```
Comment (Buzzec) [inline app/controllers/committees_controller.rb:33]: This branch includes base users, probably not what's expected

**Review by Buzzec — CHANGES_REQUESTED** [review]

File: app/controllers/committees_controller.rb (hunk 1 above)
Comment (Buzzec) [reply app/controllers/committees_controller.rb:33]: This case doesn't come up due to lack of showing in the view

**Comment by Buzzec** [comment]
There is an extra <th> in the table when logged in as a base user

**Comment by cdconn00** [comment]
> There is an extra `<th>` in the table when logged in as a base user
> https://flash-test-ft-cole-only-lnneq7.herokuapp.com/committees
> Email: [base@base.com](mailto:base@base.com)
> PW: yous*DWOB!clac0smuh

Made this change.

**Review by Buzzec — APPROVED** [review]

**Pull request merged**